docs/host-page-integration.md

```diff
@@ -0,0 +1,29 @@
+# Interacting with the sidebar
+
+## Show the sidebar:
+
+It is possible to show the Hypothesis sidebar from a third party page
+where it is embedded.
+
+To add a `button` on a page to open the sidebar, simply
+add the `data-hypothesis-trigger` attribute:
```

Comment:
Should mention that the trigger only opens the sidebar and does not close it with subsequent clicks